src/project/manifest/document.rs

```diff
@@ -259,6 +253,13 @@ impl ManifestSource {
                 }
                 Ok(())
             }
+            _ => self.add_dependency_helper(
+                name.as_source(),
+                (*requirement).clone().into(),
```

Comment:
Maybe we could use `as_deref` or `deref` instead of the * operator might read a bit easier?

Reply:
Removed the *, wasn't required after all.